scss/_modal.scss

```
@@ -159,6 +189,10 @@
     margin: $modal-dialog-margin-y-sm-up auto;
   }
 
+  .modal-dialog-scrollable {
+    height: calc(100% - (#{$modal-dialog-margin-y-sm-up} * 2));
```

This should also work and generates less css:
```suggestion
    height: calc(100% - #{$modal-dialog-margin-y-sm-up * 2});
```